vulture_os/toolkit/api_parser/carbon_black/carbon_black.py

```diff
@@ -116,7 +116,7 @@ def test(self):
                 "error": str(e)
             }
 
-    def get_logs(self, cursor=0, since=None):
+    def get_logs(self, cursor=1, since=None):
```

Comment:
Does `cursor` really starts at 1 now?

Reply:
b'{"error_code":"BAD_REQUEST","message":"start must be greater than or equal to 1"}'

Reply:
but the first call in `execute()` still sets the cursor to zéro! shouldn't that fail?